docs/mesos/vagrant-centos/units/docker.service.d/docker.conf

```diff
@@ -1,4 +1,4 @@
 [Service]
 EnvironmentFile=/etc/sysconfig/calico
 ExecStart=
-ExecStart=/usr/bin/docker daemon -H fd:// --cluster-store=etcd://${ETCD_AUTHORITY}
\ No newline at end of file
+ExecStart=/usr/bin/dockerd --cluster-store=etcd://${ETCD_AUTHORITY}
```

Comment:
So I recall that the blank execstart is actually required to "clear"  the one you are overriding. 

I did some googling to back that up, but besides a few folks doing the same, I couldn't find any definitive proof that it's necessary. 

I'd really like to restore functionality of this demo. @matthewdupre are you comfortable merging this?